geonode/groups/forms.py

```diff
@@ -52,8 +52,13 @@ def clean(self):
         cleaned_data = self.cleaned_data
 
         name = cleaned_data.get("title")
+        if not name or GroupProfile.objects.filter(title__iexact=self.cleaned_data["title"]).count() > 0:
+            raise forms.ValidationError(
+                _("A group already exists with that name."))
         slug = slugify(name)
-
+        if not slug or GroupProfile.objects.filter(slug__iexact=self.cleaned_data["slug"]).count() > 0:
```

Comment:
Django suggests in this case to use the exists and not the count:

```
Note: Don’t use this if all you want to do is determine if at least one result exists. It’s more efficient to use [exists()](https://docs.djangoproject.com/en/3.2/ref/models/querysets/#django.db.models.query.QuerySet.exists).
```

Reply:
Done